commands/root.go

```diff
@@ -112,4 +116,14 @@ func addCommands(cmd *cobra.Command, dockerCli command.Cli) {
 
 func rootFlags(options *rootOptions, flags *pflag.FlagSet) {
 	flags.StringVar(&options.builder, "builder", os.Getenv("BUILDX_BUILDER"), "Override the configured builder instance")
+
+	var timeoutDuration = defaultTimeoutCli
+	if value, ok := os.LookupEnv("BUILDX_TIMEOUT"); ok {
+		var err error
+		timeoutDuration, err = time.ParseDuration(value)
+		if err != nil {
+			timeoutDuration = defaultTimeoutCli
+		}
+	}
+	flags.DurationVar(&options.timeout, "timeout", timeoutDuration, "Override the default global timeout (20 seconds)")
```

Comment:
"(20 seconds)" not needed as already shown in default (and could be different in reality)

Reply:
Alternatively, this could take a value in seconds (assuming we don't expect users to specify timeouts in nano-seconds etc). I know we do so in some other places, for example "stop-timeout" on `docker run`;


```
     --stop-timeout int                 Timeout (in seconds) to stop a container
```

Reply:
this takes a duration at the moment, let's say we want 70 seconds timeout, then we should set it as follows:

```
--timeout 1m10s
```

if not set, we set it to `defaultTimeoutCli = 20 * time.Second` (20 seconds)

Reply:
is this okay? should we rely on timeout in seconds as an integer?